src/playlist-loader.js

```diff
@@ -264,6 +264,17 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     }).length > 1);
   };
 
+  /**
+   * Returns whether the current playlist is the final available rendition
+   *
+   * @return {Boolean} true if on final rendition
+   */
+  loader.isFinalRendition_ = function() {
+    return (loader.master.playlists.filter((playlist) => {
```

Comment:
You can just pass `loader.master.playlists.filter(isEnabled)` for the callback instead of defining an anonymous function that just calls it